app/controllers/api/v2/certificates_controller.rb

```diff
@@ -1,14 +1,23 @@
 module Api
   module V2
     class CertificatesController < ApiController
-      before_action :find_certificates
+      before_action :find_certificates, only: [:search]
 
       def search
         options = {}
         options[:include] = [:measures, 'measures.goods_nomenclature']
         render json: Api::V2::Certificates::CertificatesSerializer.new(@certificates, options.merge(serialization_meta)).serializable_hash
       end
 
+      def index
+        certificates = Certificate.actual
+                                  .eager(:certificate_descriptions)
```

Comment:
```suggestion
                                  .eager(:certificate_descriptions, :certificate_type_descriptions)
```

Reply:
Still had n+1s here

Reply:
I agree but that does not work and that's why I created that new ticket, it triggers an obscure error message not easy to fix.

Reply:
Ah that's because its singular

Reply:
```suggestion
                                  .eager(:certificate_descriptions, :certificate_type_description, :certificate_description_periods)
```

Reply:
this is still a performance issue